### PR TITLE
Change pythonanywhere instructions to use git instead of uploading fr…

### PR DESCRIPTION
**Pull request by laxdog**
…om local

## Description
I think these instructions are a bit easier to follow and cut out some of the steps that can be messed up. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] **I have read CONTRIBUTING.md**
- [x] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [x] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [x] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [N/A] **I have updated the config file accordingly if my change requires a new configuration setting or changes an existing one.**
- [N/A] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

### CODE REVIEW

**Comment by laxdog** [comment]
Should I have squashed that? Do you normally squash everything?

**Comment by rnevet** [comment]
I usually squash when there are multiple commits, keeps the master a bit cleaner.

File: docs/installation.rst [hunk 1]
```diff
@@ -97,14 +97,20 @@ You will need:
...
-#. Extract the files, and run ``lendingbot.py`` once to generate the default.cfg
-#. Modify the default.cfg with your settings (See Configuration.)
-#. Go to the "files" tab of the website.
-#. Create a new directory, preferably named "poloniexlendingbot" and upload all the files within the folder. (You cannot upload the .zip or a folder itself, you must do all the contents.)
+Downloading the bot's files to Pythonanywhere
+---------------------------------------------
+
+#. Start a new ``bash`` console from the "Consoles" tab.
+#. Get the source code from git GitHub by running ``git clone https://github.com/Mikadily/poloniexlendingbot``.
+#. You should see some output with couters increasing.
```
Comment (Evanito) [inline docs/installation.rst]: Typo: counters

Comment (laxdog) [inline docs/installation.rst]: I'm getting quite bad for spelling recently. Is there a bot that can check these for us?

**Pull request merged**